packages/storybook/src/index.ts

```diff
@@ -12,20 +12,24 @@
 // import { Component2 } from './components/component2';
 // import { OutlineAlert } from './components/shared/outline-alert/outline-alert';
 import { OutlineExample } from './components/shared/outline-example/outline-example';
-// import { OutlineCoreLink } from '@phase2/outline-core-link';
+import { OutlineCoreLink } from '@phase2/outline-core-link';
 // Add more component imports as needed...
 
 // Importing specific controllers from the `src/controllers` directory
 // import { AdoptedStylesheets } from '@phase2/outline-adopted-stylesheets-controller';
 // import { Controller1 } from './controllers/controller1';
 // import { Controller2 } from './controllers/controller2';
 // Add more controller imports as needed...
+// CSS Imports - reference css files here so that they aggregate in style.css in the 
```

Comment:
This is the comment the comment above the references for these files in `packages/storybook/index.html` is pointing people at. Helps document how files are aggregated (or not if they aren't referenced).